src/components/Semester/OrderDropdown.vue

```diff
@@ -0,0 +1,121 @@
+<template>
+  <div class="dropdown">
+    <button class="dropdown-button" data-cyId="dropdown-button">
+      <img class="dropdown-button--image" />
+      View
```

Comment:
Could this text be wrapped in a span tag and styled to look a bit more like the design? The font looks a little too light right now compared to the Figma and current "View" text on CoursePlan